packages/amplify-category-auth/src/provider-utils/awscloudformation/utils/get-app-id.ts

```diff
@@ -0,0 +1,13 @@
+import { stateManager } from 'amplify-cli-core';
```

Comment:
There's multiple code paths for getting the appId, let's not add another. I would like to see this file removed.